lib/howl/ui/text_widget.moon

```diff
@@ -96,6 +122,16 @@ class TextWidget extends PropertyObject
     @_height = height
     @gsci\set_size_request -1, height
 
+  _set_width: (width) =>
+    @_width = width
+    @gsci\set_size_request width, -1
```

Comment:
More of a FYI, but there's a property called `width_request` (and similarily `height_request`) that  should do the job when only one dimension is desired, e.g. `gsci.width_request = width`.